src/features/Main/sections/Testimonial/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC } from 'react';
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+  CarouselNext,
+  CarouselPrevious
+} from '@/components/ui/carousel';
+import { Card } from '@/components/ui/card';
+import Image from 'next/image';
+import { getAssetUrl } from '@/lib/utils';
+import { testimoni } from '@/constants/testimoni';
+
+const Testimonial: FC = () => {
+  return (
+    <div className="container mx-auto px-4 my-6 flex flex-col items-center justify-center gap-8 md:flex-row md:justify-between md:items-start">
+      <div className="">
+        <div className="flex items-center gap-4 md:gap-6 mb-4 md:mb-8">
+          <div className="w-8 md:w-16 h-1 bg-[#0F3665]"></div>
+          <h1 className="text-xl md:text-2xl font-bold">Testimonial</h1>
+        </div>
+        <h2 className="text-3xl md:text-5xl font-bold">What They Say?</h2>
+        <p className="text-md md:text-xl mt-4 md:mt-10 xl:w-5/12 lg:w-6/12 md:w-8/12">
+          Skilline has got more than 100k positive ratings from our users around
+          the world.
+        </p>
+      </div>
+      <div>
+        <Carousel
+          opts={{
+            align: 'start',
+            loop: true
+          }}
+          className="w-[15.5rem] md:w-[22rem] lg:w-[28rem]"
+        >
+          <CarouselContent>
+            {testimoni.map((items, index) => {
+              return (
+                <CarouselItem key={index}>
```

Comment:
I think better using id for mapping key, what do you think?

Reply:
so do i have to create id property in constant testimonials?

Reply:
yaps, create manual on object testimonials